orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Stage.groovy

```diff
@@ -90,7 +92,7 @@ interface Stage<T extends Execution> {
    * Gets all ancestor stages that satisfy {@code matcher}, including the current stage.
    * @return
    */
-  List<StageNavigator.Result> ancestors(Closure<Boolean> matcher)
+  List<StageNavigator.Result> ancestors(@ClosureParams(value=SimpleType, options="com.netflix.spinnaker.orca.pipeline.model.Stage, com.netflix.spinnaker.orca.batch.StageBuilder") Closure<Boolean> matcher)
```

Comment:
We should be doing this anywhere we have Closure as a method signature - especially on an abstract class or interface

(alternatively we should be using java8 functional interfaces )